### ISSUE TITLE
bump angular dependency to ^5.0.0

### ISSUE THREAD

**Issue by wkiril** [akserg/ng2-dnd#216]
* **I'm submitting a ...**
[ ] bug report
[x] feature request
[ ] question about the decisions made in the repository

Can you update the angular dependency to v5?

**Comment by akserg** [comment]
Will update on weekend.

**Comment by wkiril** [comment]
Thanks

**Comment by JohanVersteeg** [comment]
Also waiting for this. Can't update to Angular 5 at the moment. Does somebody knows a workarround?

**Comment by johnrazmus** [comment]
Is there an updated eta for when there's time to update this library?

**Comment by JohnArrowwood** [comment]
+1

My project was already upgraded to Angular 5 due to another dependency that demanded it.  When I tried to add drag-and-drop support, it wouldn't work, and it was not obvious why.  None of the drag-and-drop functionality works, and there is nothing in the console.log().  

Eventually, I tried creating a new project from scratch, and it still didn't work, which prompted me to come here, with intent to report it.

Is there a work-around while waiting for a fix?
